controller/emailVerify.js

```diff
@@ -0,0 +1,31 @@
+const nodemailer = require('nodemailer')
+const path = require('path');
+var EmailTemplate = require('email-templates');
+exports.verifyEmail = async (email, name, host, token) => {
+    let transporter = nodemailer.createTransport({
```

Comment:
the 'transporter' variable, you have defined, looks like it can be defined as const, is there a reason you used let?

Reply:
changed to cosnt